phase-6/submissions/transfer-dtag/EdouardLvdl

```diff
@@ -0,0 +1 @@
+EE5CAB3749CC395FEB0A71245523D181534FD5E00B4186FCDAE6114B8F1CD1D2
```

Comment:
Looks like you forgot to add the `MsgRequestDTagTransfer` transaction hash. That is required

Reply:
Missed the notification, finally updated the tx hashes! Thanks for reviewing